ex21.py

```diff
@@ -2,30 +2,33 @@ def add(a, b):
 	print "ADDING %d + %d" % (a, b)
 	return a + b
 
-# creating the function "subtract()" which takes the arguments a and b.
+# the function "subtract()" takes the numeric arguments a and b.
 # "subtract()" then prints, "SUBTRACTING" followed by the integer value
 # of a, then a "-", followed by the integer value of b. Finally,
-# the function returns the difference of a and b
+# the function returns the [possibly floating point] difference of a and b
 def subtract(a, b):
 	print "SUBTRACTING %d - %d" % (a, b)
 	return a - b
 
+# like subtract(), but with the arithmetic product of a and b.
 def multiply(a, b):
 	print "MULTIPLYING %d * %d" % (a, b)
 	return a * b
 
+# like multiply, but with the arithmetic division of a and b
 def divide(a, b):
 	print "DIVIDING %d / %d" % (a, b)
 	return a / b
 
 print "Let's do some math with just functions!"
 
-# playing with floating points. %d only returns integers without rounding

```

Comment:
"%d" is the format string for *printing* numbers as integers (without the decimal point), not returning numbers.